### PR TITLE
Adding coverage for Registry entity in WebUI

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/ui/test_docker.py:DockerRegistriesTestCase
......
----------------------------------------------------------------------
Ran 6 tests in 283.862s

OK
```

### CODE REVIEW

**Comment by elyezer** [comment]
Can you please add the new added module to the documentation?

ACK, pending adding to docs.

**Comment by oshtaier** [comment]
Done. Thanks @elyezer for pointing out on that

**Comment by oshtaier** [comment]
All tests related to new entity placed into test_docker file, so updated only one documentation file as we do not have new test_registry module

File: robottelo/ui/factory.py [hunk 1]
```diff
@@ -663,3 +664,22 @@ def make_oscappolicy(session, **kwargs):
     page = session.nav.go_to_oscap_policy
     core_factory(create_args, kwargs, session, page)
     OpenScapPolicy(session.browser).create(**create_args)
+
+
+def make_registry(session, org=None, loc=None, force_context=True, **kwargs):
+    """Creates a Registry"""
+    create_args = {
+        u'name': None,
```
Comment (abalakh) [inline robottelo/ui/factory.py]: bonus karma for sorting alphabetically :)

File: tests/foreman/ui/test_docker.py [hunk 2]
```diff
@@ -1338,19 +1379,39 @@ def test_update_registry_username(self):
...
+        with Session(self.browser) as session:
+            for name in valid_data_list():
+                with self.subTest(name):
+                    make_registry(
+                        session,
+                        name=name,
+                        url=gen_url(subdomain=gen_string('alpha')),
+                        description=gen_string('utf8'),
+                    )
+                    self.registry.delete(name)
```
Comment (abalakh) [inline tests/foreman/ui/test_docker.py]: no validation that registry actually disappeared from the list?

**Comment by abalakh** [comment]
ACK pending non-blocking nitpicks

File: tests/foreman/ui/test_docker.py (hunk 2 above)
Comment (oshtaier) [inline tests/foreman/ui/test_docker.py]: That is actually result of my delete functionality re-factor procedure. @sghai mentioned that assertion should be always performed on test level, but it is pretty hard to implement that approach without huge amount of redundant code
You can refer to https://github.com/SatelliteQE/robottelo/pull/2809 for more details about improvement

**Comment by oshtaier** [comment]
@abalakh Comment addressed. Please, merge in case you okay with PR

**Pull request merged**